packages/calling-stateful-client/src/StatefulCallClient.test.ts

```diff
@@ -55,14 +55,14 @@ jest.mock('@azure/communication-calling', () => {
       };
     }),
     Features: {
-      get Recording(): CallFeatureFactoryType<RecordingCallFeature> {
-        return MockRecordingCallFeatureImpl;
+      get Recording(): CallFeatureFactory<RecordingCallFeature> {
+        return { callApiCtor: MockRecordingCallFeatureImpl };
       },
-      get Transcription(): CallFeatureFactoryType<TranscriptionCallFeature> {
-        return MockTranscriptionCallFeatureImpl;
+      get Transcription(): CallFeatureFactory<TranscriptionCallFeature> {
+        return { callApiCtor: MockTranscriptionCallFeatureImpl };
```

Comment:
I am curious if we should call this callApiCtr or callFeatureCtor?

Reply:
Good point, that's feedback for the headless API: 
![image](https://user-images.githubusercontent.com/82062616/142289708-9e9aafd4-5aa0-4e12-aa6d-0faa07c25021.png)